.env

```diff
@@ -1 +1,2 @@
 GENERATE_SOURCEMAP=false
+FAST_REFRESH=false
```

Comment:
We lose SourceMaps and fast refresh by setting this.

Need to research a suitable workaround that allows for both of these very nice, previously-working features